test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -431,7 +400,7 @@ Describe "Validate Update-Help -SourcePath for all PowerShell modules for user s
         $ProgressPreference = $SavedProgressPreference
     }
 
-    RunUpdateHelpTests -Tag "Feature" -useSourcePath -Scope 'CurrentUser'
+    RunSaveHelpTests -Tag "Feature" -useSourcePath -UserScope
```

Comment:
I don't mind the "style" changes in the rest of the PR, they look like they helped with debugging.

However, here's an actual bug. This test is supposed to validate `Update-Help` (which takes a `-UserScope` switch) but was erroneously changed to `RunSaveHelpTests` (which is covered by two tests below).
```suggestion
    RunUpdateHelpTests -Tag "Feature" -useSourcePath -UserScope
```